CompuCell3D/core/CompuCell3D/Boundary/BoundaryStrategy.cpp

```diff
@@ -91,17 +107,17 @@ BoundaryStrategy::~BoundaryStrategy() {
     CC3D_Log(LOG_DEBUG) << "strategy_z=" << strategy_z;
     if (strategy_x) {
         delete strategy_x;
-        strategy_x = NULL;
+        strategy_x = nullptr;
     }
 
     if (strategy_y) {
         delete strategy_y;
-        strategy_y = NULL;
+        strategy_y = nullptr;
     }
 
     if (strategy_z) {
         delete strategy_z;
-        strategy_z = NULL;
+        strategy_z = nullptr;

```

Comment:
Nice. This seems to be the correct way since NULL is an integer whereas nullptr is a ptr.